enhancements/cluster-logging/forwarding-json-structured-logs.md

```diff
@@ -28,97 +28,73 @@ see-also:
 
-```json
-{
-  "structured":{
-    "name":"fred",
-    "home":"bedrock",
-  }",
-  "kubernetes":{"labels":{"app.kubernetes.io/name": "flintstones", ...}},
-  ...
-}
+``` json
+{"message":"{\"level\":\"info\",\"name\":\"fred\",\"home\":\"bedrock\"",
+ "more fields..."}
 ```
 
-This proposal describes
-
-* extensions to the logging [data model][data_model] - `structured` field.
-* extensions to the `ClusterLogForwarder` API to configure JSON parsing and forwarding.
-* extensions to the `ClusterLogForwarder` API to select an index.
-* indexing structured records in current and future Elasticsearch stores.
-* replacing the [defunct MERGE_JSON_LOG][defunct_merge] feature.
-
-**Note**: This proposal focuses on JSON, but the data model and API changes can apply to other structured formats that may be supported in future.
+The proposed new record with JSON parsing enabled looks like this:
 
-## Terminology
-
-Logging terminology can have overlapping meanings, this document uses the following terms with specific meanings:
-
-- *Consumer*: A destination for logs, identified by a forwarder `output`.
-  The default consumer is the *Elasticsearch Log Store*
-- *Entry*: a single log entry, usually a single line of text. May be JSON or other format.
-- *Structured Log*: a log where each *entry* is formatted as a *JSON object*.
-- *Record*: A key-value record including the *entry* and meta-data collected by the logging system.
-- *Data Model*: [Description][data_model] of the forwarder's own *record* format.
-
-Elasticsearch *Index*: an index receives a stream of logs with similar JSON format.
+``` json
+{"structued": { "level": "info", "name": "fred", "home": "bedrock" },
+ "more fields..."}
+```
 
 ## Motivation
 
 ### Goals
 
-* Direct access to JSON log entries as JSON objects for `fluentdForward`, `Elasticsearch` or any other JSON-aware consumer.
-* Elasticsearch outputs can associate index names with log records by category, namespace, k8s label, or`input` selection criteria.
-* Upgrade path from today's implementation.
+* Direct access to JSON log entries as JSON objects
+  * for `fluentdForward`, `Elasticsearch` or any other JSON-aware consumer.
+* Elasticsearch outputs can direct logs with different JSON formats to different indices based on category, k8s label, or input selectors.
+* Backwards compatible with today's implementation.
 
 ### Non-Goals
 
-* General-purpose JSON queries and transformations.
+* Not general-purpose JSON queries and transformations.
+* No improved flexibility of the ES output for external ES deployments.
 
 ## Proposal
 
 ### Data model
 
-One new top-level fields added to the logging output record [data model][data_model]:
+Two top-level fields added to the logging output record [data model][data_model]:
 
 * `structured` (type object): Original structured JSON log entry.
+* `elasticsearch` (type object) Fields used only by an Elasticsearch output:
+  * `index`: Name of an elasticsearch index
```

Comment:
out of curiosity, what does this buy us?
there already is a means to denote the intended index and this feels like extra data to add in that we aren't going to use.

Reply:
Suggesting this as a more consistent name to replace the existing "viaq_elasticsearch_index". If we leave viaq_elasticsearch_index in place then we will have to document it to support unmanaged elasticsearch instances. 
I'm ok with dropping this and keeping viaq_... given the time constraints.